bindings/python/native/src/client/high_level_api.rs

```diff
@@ -163,6 +165,22 @@ impl Client {
         })?;
         Ok(children.iter().map(|child| hex::encode(child.as_ref())).collect())
     }
+    /// Get the message id from the payload string.
+    ///
+    /// Args:
+    ///     payload_str (str): The identifier of message.
+    ///
+    /// Returns:
+    ///     children ([str]): The returned list of children string.
```

Comment:
Should this be `message_id (str): The identifier of message.`?

Reply:
Yes. Thanks for identifying this.